data/snippets/yaml.md

```diff
@@ -0,0 +1,28 @@
+```
+$ echo "---
+dict :
+  key:
+        value
+" >> test.yaml
+$ coala --files test.yaml --bears YAMLLintBear
```

Comment:
@yukiisbored we don't add this by design so people do see the warning about the `--save` flag, it's a very conscious part to make the learning curve of coala easier for new users